shared-ui/components/meet-the-team/meet-all-core/MeetAllCore.styles.ts

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import { P, H3 } from '../../../style/typography';
+import { colors } from '../../../style/colors';
+import { max } from '../../../lib/responsive';
+
+const StyledMeetCoreContainer = styled.div`
+  display: flex;
+  margin: auto;
+  padding: 8em;
+  @media ${max.tabletLg} {
+    padding: 4em;
+  }
+`;
+
+const StyledOurTeamCore = styled(P)`
+  color: ${colors.WHITE};
+  text-align: left;
+  padding-top: 3.5em;
+`;
+
+const StyledHeaderCore = styled(H3)`
+  text-align: left;
+`;
+
+const StyledDescriptionTeamCore = styled(P)`
+  color: ${colors.TEXT_BROWN};
+  text-align: left;
+  padding-top: 1em;
+`;
+
+const StyledImageCore = styled.img`
+  right: 5em;
+  margin: auto;
+  width: 100%;
+  @media ${max.tabletLg} {
+    width: 55%;
+  }
+  @media ${max.tabletLg} {
+    width: 70%;
+  }
+  @media ${max.tabletSm} {
+    width: 100%;
+  }
+`;
```

Comment:
Can we change the queries to be something like this so it's mostly 100% the entire time except when it looks a bit too big for the mobile design
```
const StyledImageCore = styled.img
  right: 5em;
  margin: auto;
  width: 100%;
  @media ${min.tablet} {
    width: 90%;
  }
  @media ${min.tabletLg} {
    width: 100%;
  }
`;
```